.buildkite/build.rayci.yml

```diff
@@ -0,0 +1,39 @@
+group: build
+steps:
+  - label: ":tapioca: build: wheel {{matrix}}"
+    instance_type: medium
+    commands:
+      - ./ci/build/build-manylinux-ray.sh
```

Comment:
maybe put these into separate scripts so that it is not encoded on the pipeline?

Reply:
totally